src/AWS/Orleans.Streaming.SQS/Streams/SqsStreamOptions.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections.Generic;
+using Microsoft.Extensions.Options;
+
+namespace Orleans.Hosting
+{
+    public class SqsStreamOptions : PersistentStreamOptions
+    {
+        public string DeploymentId { get; set; }
+
+        public string DataConnectionString { get; set; }
```

Comment:
Since this is provider-specific, could we call this `ConnectionString` instead?

Reply:
I think it should not be standardazied as connection string on all providers... others don't have it but instead, they ask for something like endpoint and API key... for those cases, our options should reflect that with 2 properties just like I did with the CosmosDB providers.

This `DataConnectionString` there was a hack back in the day just to "follow the standard". 

I'll submit a PR to @jason-bragg branch changing that as well

Reply:
Yes, specific names per-provider is fine. Whatever is the typical lingo for that system is what we should use.